client/web/src/lib/ColorPicker.ts

```diff
@@ -0,0 +1,46 @@
+interface ColorPickerOptions {
+  el: Element;
+};
+
+export interface RGBAColor {
+  r: number;
+  g: number;
+  b: number;
+  a?: number;
+};
+
+export class ColorPicker {
+  constructor(options: Partial<ColorPickerOptions> = {}) {
+
+  }
+
+  setColor(color: string | number | RGBAColor) {
+
+  }
+
+  getRGB() {
+    return { r: 1, g: 1, b: 1, };
```

Comment:
Trailing comma only after multiple-line properties in an object, not in single-line objects.